addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -133,7 +133,11 @@ export class Powerbox {
         const selection = document.getSelection();
         if (!selection.isCollapsed || !selection.rangeCount) return;
 
-        if (event.key === '/' && !this._active) {
+        if (
+            event.key === '/' &&
+            !this._active &&
+            (!this.options.shouldActivate || this.options.shouldActivate())
```

Comment:
In which case could we not want the `shouldActivate` option to even exist?

Reply:
I don't think we have a case but API wise, I don't see why we should enforce to specify the method in the options.
By default the powerbox is active upon key '/' down. Only if we wish to prevent that behavior, we specify an option.

Reply:
Sure indeed, I was just inquiring :-)